### PR TITLE
Update Main Page

### LINKED ISSUE #318: Add in Direct Donation Via Paypal
Using this link :  https://paypal.me/faforever set on the website to allow people to use direct donations rather than pateron

### PR DESCRIPTION
**Pull request by MrRowey**
Updated Some of the text on the main FAF Page.

### CODE REVIEW

File: templates/views/index.pug [hunk 1]
```diff
@@ -80,8 +80,8 @@ block content
                 p Unleash the most powerful weapon in the galaxy, the Supreme Commander, in one of four diverse factions: The United Earth Federation, The Cybran Nation, The Aeon Illuminate or The Order (Seraphim).
         .row.text-center.buy-now
             hr
-            h3 In order to play Forged Alliance Forever, you must first install Supreme Commander: Forged Alliance.  You can purchase it for less than 10€, below:
-            .col-md-12
+            h3 To play Forged Alliance Forever, you must first install Supreme Commander: Forged Alliance on Steam.  You can purchase it for less than 10€, below:
```
Comment (Brutus5000) [inline templates/views/index.pug]: Please leave out steam as we intend to implement gog linking and then we'll forget to touch it again

File: templates/views/index.pug [hunk 2]
```diff
@@ -20,16 +20,16 @@ block content
     .the-project: .container
         .col-md-3
         .col-md-6
-            h1 The Project
-            p Forged Alliance Forever is a community-driven project designed to facilitate online play for Supreme Commander: Forged Alliance. Complete with chat, matchmaking, replay vaults, tournaments and ever evolving balance.
+            h1 What is Forged Alliance Forever?
+            p Forged Alliance Forever is a community-driven project designed to facilitate online play for Supreme Commander: Forged Alliance. Complete with chat, matchmaking, replay vaults, tournaments, Map & Mod Vault and ever-evolving balance.
```
Comment (Brutus5000) [inline templates/views/index.pug]: should we stick to the same capitalisation rules (keep lower case)?

Comment (MrRowey) [reply templates/views/index.pug]: Sure will alter this to all lowercase

File: templates/views/index.pug (hunk 1 above)
Comment (MrRowey) [reply templates/views/index.pug]: That's a fair point. Do we know how long till we support GOG ?

Comment (Brutus5000) [reply templates/views/index.pug]: As soon as you finish the PR #291 ? :stuck_out_tongue_closed_eyes:

**Review by Brutus5000 — APPROVED** [review]

**Comment by Brutus5000** [comment]
@MrRowey Since you marked it as draft I'm not going to merge it.If you merge it yourself please make sure to select squash&merge

**Pull request merged**